packages/apollo-codegen-swift/src/codeGeneration.ts

```diff
@@ -420,15 +427,20 @@ export class SwiftAPIGenerator extends SwiftGenerator<CompilerContext> {
         }
 
         this.printNewlineIfNeeded();
-        this.printOnNewline("public static let possibleTypes = [");
+        this.printOnNewline(swift`public static let possibleTypes = [`);
         this.print(
-          join(variant.possibleTypes.map(type => `"${type.name}"`), ", ")
+          join(
+            variant.possibleTypes.map(
+              type => swift`${SwiftSource.string(type.name)}`
+            ),
+            ", "
```

Comment:
Am I correct that this doesn't need `SwiftSource.string` since as the separator parameter, it needs to be a string literal?

Reply:
Yeah. I put a note to that effect on the doc comment for this function; I left the separator as `string` for convenience as it's expected to just be something like `", "`. I originally left the start/end strings for `wrap` as that too, until I discovered a call to `wrap` that used string interpolation in the `start` parameter. Having the separator to `join` be ` string` is technically unsafe because you could interpolate something into it, but I thought the convenience outweighed that here as I can't think of a non-contrived scenario where I'd interpolate an identifier into the separator.